src/app/screens/BookList/index.tsx

```diff
@@ -1,18 +1,22 @@
 import React from 'react';
-import { View } from 'react-native';
+import { FlatList, ListRenderItem } from 'react-native';
 import BookCard from '@components/BookCard';
+import { BOOKS } from '@constants/mockData';
+import { Book } from '@interfaces/book';
 
 import styles from './styles';
 
+export const keyExtractor = ({ id }: Book) => String(id);
+export const renderBook: ListRenderItem<Book> = ({ item }) => <BookCard bookData={item} />;
```

Comment:
Sorry but this is still wrong, please check the previous comments.
Why is this still outside the function declaration?
Also => `const renderBook: ListRenderItem<Book> = ({ item }) => <BookCard {...item} />;`

Reply:
Yeat. I missed up.